lib/Listener/DataSetDataProviderListener.php

```diff
@@ -169,6 +169,7 @@ private function getData(DataSet $dataSet, DataProviderInterface $dataProvider):
             // Start should be the lower limit
             // Size should be the distance between upper and lower
             $filter['start'] = $lowerLimit;
+            $filter['end'] = $upperLimit;
```

Comment:
I am not clear on what this line here is for.

Reply:
I think this was just to try to fix the limits not working and I left it there, I'm going to remove it. Good catch! :)